.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: '<what happen when you do on which document project>'
+labels: 'bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+```
+<Paste your command-line here which cause the problem>
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Your project**
+Link to your sphinx project, or attach zipped small project sample.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment info**
+- OS: [e.g. Unix/Linux/Mac/Win/other with version]
+- Python version: [e.g. 3.7.1]
+- Sphinx version: [e.g. 1.8.2]
+- Extra tools: [e.g. Browser, tex or something else]
```

Comment:
Usually, issuers use 3rd party extensions. So "extensions" information is also needed.

Reply:
Updated: https://github.com/sphinx-doc/sphinx/pull/5942/files#diff-74c2649c999338ed7a2833d79d4b25c3R39